src/core/blocks/strings.cpp

```diff
@@ -133,6 +133,55 @@ struct Replace : public Common {
   }
 };
 
+struct Join {
+  static CBTypesInfo inputTypes() { return CoreInfo::StringSeqType; }
+  static CBTypesInfo outputTypes() { return CoreInfo::StringType; }
+
+  static CBParametersInfo parameters() { return CBParametersInfo(params); }
+
+  void setParam(int index, const CBVar &value) {
+    switch (index) {
+    case 0:
+      _separator = value.payload.stringValue;
+      break;
+    default:
+      break;
+    }
+  }
+
+  CBVar getParam(int index) {
+    switch (index) {
+    case 0:
+      return Var(_separator);
+    default:
+      return Var::Empty;
+    }
+  }
+
+  CBVar activate(CBContext *context, const CBVar &input) {
+    if (input.payload.seqValue.len == 0)
+      return Var::Empty; // FIXME or Var("")?
+
+    std::ostringstream ss;
+    ss << input.payload.seqValue.elements[0].payload.stringValue;
+
+    for (uint32_t i = 1; i < input.payload.seqValue.len; i++) {
+      assert(input.payload.seqValue.elements[i].valueType == String);
+      auto str = input.payload.seqValue.elements[i].payload.stringValue;
+      ss << _separator << str;
+    }
+
+    CBVar output{};
+    ::chainblocks::cloneVar(output, Var(ss.str()));
```

Comment:
oh wow no, with buffer you won't need this either.. just pass
`return `Var(_buffer)`

Reply:
again check (AppendTo)